tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "incremental": true,
     "jsx": "react",
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "bundler",
```

Comment:
This change is required to be able to build the settings for `Ollama`.

It seems that the `"moduleResolution": "node",` is deprecated anyway, and `"bundler"` seems to be a replacement https://www.typescriptlang.org/tsconfig/#moduleResolution.

Reply:
What is the TypeScript issue otherwise?

Reply:
The issue is only when building the settings. It uses `ts-json-schema-generator`, which itself requires a `tsconfig` file.
Actually, the simple/safer way is probably to have separate tsconfig.

Reply:
> What is the TypeScript issue otherwise?

:smile: I was not expected it after the last commit, but here is the error https://github.com/jupyterlite/ai/actions/runs/15013843059/job/42187366202?pr=69#step:6:73

I think that it is related to `mjs` files that can be imported...

Reply:
So it also fails at build time, when importing `@langchain/ollama`, which import `ollama/browser`
```ts
import { Ollama } from "ollama/browser";
```

Here is the typescript error, about importing types from `mts` files :
```
Cannot find module 'ollama/browser' or its corresponding type declarations.
  There are types at '/home/brichet/projects/jupyterlite-ai/node_modules/ollama/dist/browser.d.mts', but this result could not be resolved under your current 'moduleResolution' setting. Consider updating to 'node16', 'nodenext', or 'bundler'.ts(2307)
```

Reply:
I'll have a quick look locally.

Reply:
The relevant issue appears to be https://github.com/ollama/ollama-js/issues/151

Reply:
Maybe we can use the `paths` in `tsconfig.json` for the `ollama/browser` import:

```json
"paths": {
  "ollama/browser": ["./node_modules/@langchain/ollama/dist"]
},
```

Reply:
https://github.com/brichet/jupyterlite-ai/pull/1 appears to be building fine locally.

cc @brichet in case you would like to have a look